Resources/Prototypes/_RMC14/Roles/Loadouts/Items/RoleSpecific/synthetic.yml

```
@@ -0,0 +1,299 @@
+# Synthetic
+
+# UNMC Jumpsuits
+- type: loadout
+  id: ScrubsBlueRMC
+  equipment:
+    jumpsuit: CMScrubsBlue
+
+- type: loadout
+  id: ScrubsLightBlueRMC
+  equipment:
+    jumpsuit: RMCScrubsLightBlue
+
+- type: loadout
+  id: ScrubsGreenRMC
+  equipment:
+    jumpsuit: CMScrubsGreen
+
+- type: loadout
+  id: ScrubsPurpleRMC
+  equipment:
+    jumpsuit: CMScrubsPurple
+
+- type: loadout
+  id: ScrubsOliveRMC
+  equipment:
+    jumpsuit: RMCScrubsOlive
+
+- type: loadout
+  id: ScrubsGreyRMC
+  equipment:
+    jumpsuit: RMCScrubsGrey
+
+- type: loadout
+  id: ServiceUniformRMC
+  equipment:
+    jumpsuit: CMJumpsuitBO
+
+- type: loadout
+  id: ServiceUniformXORMC
+  equipment:
+    jumpsuit: CMJumpsuitXOFormal
+
+- type: loadout
+  id: ComTechUniformRMC
+  equipment:
+    jumpsuit: CMJumpsuitMarineEngineer
+
+- type: loadout
+  id: EngineerUniformRMC
+  equipment:
+    jumpsuit: CMJumpsuitMaintTech
+
+- type: loadout
+  id: MPUniformRMC
+  equipment:
+    jumpsuit: CMJumpsuitMP
+
+- type: loadout
+  id: DCCJumpsuitRMC
+  equipment:
+    jumpsuit: CMJumpsuitDCC
+
+# Civilian Jumpsuits
+- type: loadout
+  id: TShirtWhiteRMC
+  storage:
+    back:
+    - CMJumpsuitTShirtWhite
+
+- type: loadout
+  id: TShirtGrayRMC
+  storage:
+    back:
+    - CMJumpsuitTShirtGray
+
+- type: loadout
+  id: TShirtRedRMC
+  storage:
+    back:
+    - CMJumpsuitTShirtRed
+
+- type: loadout
+  id: CivilianJumpsuitRMC
+  storage:
+    back:
+    - RMCJumpsuitCivilian
+
+- type: loadout
+  id: WhiteServiceJumpsuitRMC
+  storage:
+    back:
+    - RMCJumsuitWhiteService
+
+- type: loadout
+  id: LiaisonBlueJumpsuitRMC
+  storage:
+    back:
+    - CMJumpsuitLiaisonBlue
+
+- type: loadout
+  id: LiaisonBrownJumpsuitRMC
+  storage:
+    back:
+    - CMJumpsuitLiaisonBrown
+
+- type: loadout
+  id: LiaisonCorporateFormalJumpsuitRMC
+  storage:
+    back:
+    - CMJumpsuitLiaisonCorporateFormal
```

```suggestion
  equipment:
    jumpsuit: CMJumpsuitTShirtWhite

- type: loadout
  id: TShirtGrayRMC
  equipment:
    jumpsuit: CMJumpsuitTShirtGray

- type: loadout
  id: TShirtRedRMC
  equipment:
    jumpsuit: CMJumpsuitTShirtRed

- type: loadout
  id: CivilianJumpsuitRMC
  equipment:
    jumpsuit: RMCJumpsuitCivilian

- type: loadout
  id: WhiteServiceJumpsuitRMC
  equipment:
    jumpsuit: RMCJumsuitWhiteService

- type: loadout
  id: LiaisonBlueJumpsuitRMC
  equipment:
    jumpsuit: CMJumpsuitLiaisonBlue

- type: loadout
  id: LiaisonBrownJumpsuitRMC
  equipment:
    jumpsuit: CMJumpsuitLiaisonBrown

- type: loadout
  id: LiaisonCorporateFormalJumpsuitRMC
  equipment:
    jumpsuit: CMJumpsuitLiaisonCorporateFormal
```
Forgot to do it with these